pages/api/index.ts

```diff
@@ -0,0 +1,18 @@
+import { prisma } from "@/libs/prisma";
+const handler = async (req,res) => {
+    if(req.method==='POST'){
+        console.log(req.body.toShort);
+        const ra = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ1234567890"
```

Comment:
Use more viable variable name instead to `ra`

Reply:
Hey @RahulNavneeth , 
    Made the concurrent changes you proposed 😄